packages/website/src/shared/lib/utilities/enum.test.ts

```diff
@@ -0,0 +1,38 @@
+import _ from "lodash";
+import { getFirst, getNth } from "./enum";
+
+enum MOCK_ENUM {
+  FF = "Final Fantasy",
+  FNAF = "Five Nights at Freddy's",
+  GTA = "Grand Theft Auto",
+  KH = "Kingdom Hearts",
+  LOL = "League of Legends",
+  OW = "Overwatch",
+  VAL = "Valorant",
+}
+
+enum MOCK_EMPTY_ENUM {}
+
+describe("Enum Utilities", () => {
+  describe("getNth", () => {
+    it("gets the nth enum", () => {
```

Comment:
Not sure if this is intended effect. Tried some things out on coderpad with codebelow

```
import { chain, nth, values, each, size, range } from "lodash";

export const getNth = (enums: Record<any, any>, index: number) => chain(enums).values().nth(index).value();

enum Test {
  a = 1,
  b = 2,
  c = 3
}

console.log(Test)
// { '1': 'a' , '2': 'b', '3': 'c', a: 1, b:2, c:3}


each(range(0, size(Test)), (n) => {
  console.log(n)
});

Output:
0
1
2
3
4
5
6

```

Found out, [enums transpiling number values](https://stackoverflow.com/questions/56044872/typescript-enum-object-values-return-value) by Typescript creates 2x size of enum. While test passes currently, the above quirk is like an edge case.

Reply:
That's good to keep note of. Though, I don't think this happens with the test, since we use string enums.

Mock enum output:
```
console.log(MOCK_ENUM);

console.log
    {
      FF: 'Final Fantasy',
      FNAF: "Five Nights at Freddy's",
      GTA: 'Grand Theft Auto',
      KH: 'Kingdom Hearts',
      LOL: 'League of Legends',
      OW: 'Overwatch',
      VAL: 'Valorant'
    }
```

Test with log:
```
_.each(_.range(0, _.size(MOCK_ENUM)), (n) => {
  console.log("n:", n, "Received:", getNth(MOCK_ENUM, n), "Expected:", _.nth(_.toArray(MOCK_ENUM), n));
  expect(getNth(MOCK_ENUM, n)).toStrictEqual(_.nth(_.toArray(MOCK_ENUM), n));
});
```

Test output:
```
    n: 0 Received: Final Fantasy Expected: Final Fantasy
    n: 1 Received: Five Nights at Freddy's Expected: Five Nights at Freddy's
    n: 2 Received: Grand Theft Auto Expected: Grand Theft Auto
    n: 3 Received: Kingdom Hearts Expected: Kingdom Hearts
    n: 4 Received: League of Legends Expected: League of Legends
    n: 5 Received: Overwatch Expected: Overwatch
    n: 6 Received: Valorant Expected: Valorant
```